hugo/config.toml

```diff
@@ -2,6 +2,7 @@ languageCode = "en-us"
 title = "ACL Anthology"
 disablePathToLower = true
 staticDir = ["static", "data-export"]
+rssLimit = 1000
```

Comment:
I wonder what's a good limit here. A single major conference exceeds this already; e.g. ACL 2023 had ~1300 papers, not even counting workshops.

Reply:
Right, a cutoff by date might be ideal. Or even better, we include at least N items (e.g. 1000), but make sure to include all items from the same date as the oldest one. But if that's too complex to implement we can also just go for an arbitrary number, too :)

Reply:
I think we can do something with the [`where` function](https://gohugo.io/functions/collections/where/) here. I'll take a closer look at that tomorrow.

Reply:
> Or even better, we include at least N items (e.g. 1000), but make sure to include all items from the same date as the oldest one.

I feel like I strained the templating engine to the limits of its expressiveness here, but I think 115815a should implement that behavior. I couldn't test it with > 1000 pages on my local machine yet because I drastically reduced the data folder contents to speed up the builds.

Reply:
Ah nice, that looks good on first glance!